src/main/java/org/jabref/model/pdf/search/SearchFieldConstants.java

```diff
@@ -10,5 +10,5 @@ public class SearchFieldConstants {
 
     public static final String[] PDF_FIELDS = new String[]{PATH, CONTENT, PAGE_NUMBER, MODIFIED, ANNOTATIONS};
 
-    public static final String VERSION = "0.5a";
+    public static final String VERSION = "lucene91";
```

Comment:
The resulting path is then (on Windows, for me)

`C:\Users\koppor\AppData\Local\org.jabref\JabRef\lucene91`

Is IMHO OK as the path changes "only" of Lucene is updated, not if JabRef is updated.

Reply:
The idea of the version string is to make sure to update the index if something changes in JabRefs handling, i.e. a field is added or something or JabRef switches to a newer version of lucene and breaks compatibility somehow. Therefore, I would opt not to tie it to a lucene version number, but continue counting up JabRefs version (could change to 0.6a and once the overall switch to lucene is complete we could change to v1.0).

Reply:
The thing is that Lucene cannot read or write other index versions if no compatibility module is loaded. Currently, such a compatibility module leads to an error during jlink: 64kb Limit 🙈

Reply:
It can read the older index, but cannot write to it since it's older

Reply:
Well, currently we don't use that anyway. If the version in that string is changed, the index is discarded and rebuilt. We would only need to consistently change that version string on every lucene version update.